pygyat/__init__.py

```diff
@@ -39,5 +39,17 @@
     "sigma": ">",
     "beta": "<",
     "diddy": "in",
-    "pluh": "pass"
+    "pluh": "pass",
+    "is":"oil\s+up",
```

Comment:
This should be the other way around, no?

Reply:
That's right. changing it rn.

Reply:
> That's right. changing it rn.

-1000000 aura